testsuites/syncgateway/performance/run_gateload_perf_test.py

```diff
@@ -200,7 +218,9 @@ def run_gateload_perf_test(gen_gateload_config, test_id, gateload_params):
         feed_type=opts.feed_type,
         sleep_time_ms=opts.sleep_time_ms,
         channel_active_users=opts.channel_active_users,
-        channel_concurrent_users=opts.channel_concurrent_users
+        channel_concurrent_users=opts.channel_concurrent_users,
+        delay_profiling_secs=opts.delay_profiling_secs,
+        delay_expvar_collect_secs=opts.delay_expvar_collect_secs
```

Comment:
`delay_profiling_secs ` and `delay_expvar_collect_secs` should be passed into the scenario def  `run_gateload_perf_test` rather than `GateloadParams`. I think we should reserve `GateloadParams` for anything related to the parametrized Gateload config.